index.html

```diff
@@ -78,7 +78,7 @@ <h1 class="name">I'm Milen Glad <br />To See You!</h1>
         </li>
       </ul>
     </section>
-
+    <div class="project-container hide"></div>
     <section class="card-container" id="portfolio">
       <section class="card">
         <img src="./images/card1.png" class="cardpic" alt="" />
```

Comment:
- [ ] The projects section should be created dynamically with the information from the projects array, kindly implement this change so your project has all the required functionality

Reply:
@Ismailco , can you please elaborate?
I have used an array to store all the information for the pop-up project section in JavaScript and it opens different cards dynamically with the given pictures and content. Today is the 5th day redoing everything would take time please take this into consideration.

Reply:
@milen-ale The project section should be created using JavaScript, you need to remove the project section from your HTML file and inject it from the js file to the DOM (to the page). the popup is working fine